Ray worker pid: 27739
/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/numpy/core/fromnumeric.py:3118: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/numpy/core/_methods.py:85: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Traceback (most recent call last):
  File "/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/ray/workers/default_worker.py", line 98, in <module>
    ray.worker.global_worker.main_loop()
  File "/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/ray/worker.py", line 1052, in main_loop
    self._wait_for_and_process_task(task)
  File "/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/ray/worker.py", line 1003, in _wait_for_and_process_task
    self._process_task(task, execution_info)
  File "/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/ray/worker.py", line 908, in _process_task
    self.actors[key], *arguments)
  File "/home/beast/anaconda3/envs/srikar/lib/python3.6/site-packages/ray/function_manager.py", line 792, in actor_method_executor
    method_returns = method(actor, *args)
  File "/home/beast/srikar/apex/rl/algos/td3.py", line 187, in collect_experience
    new_obs, reward, done, _ = self.env.step(action)
  File "/home/beast/srikar/apex/cassie/speed_double_freq_env.py", line 143, in step
    self.step_simulation(action)
  File "/home/beast/srikar/apex/cassie/speed_double_freq_env.py", line 93, in step_simulation
    target = action + ref_pos[self.pos_idx]
KeyboardInterrupt
